msg/ko_KR.euckr/csql.msg

```diff
@@ -161,7 +161,9 @@ $ csql help messages
    ;COMmit                      - ���� ���� Ʈ����� Ŀ��.\n\
    ;ROllback                    - ���� ���� Ʈ����� öȸ.\n\
    ;AUtocommit [ON|OFF]         - �ڵ� Ŀ�� ��� ���|���.\n\
-   ;REStart                     - �����ͺ��̽� �����.\n\
+   ;REStart                     - CSQL ¼¼¼�<87>¿¡¼ �<87>ö�<80>ç µ¥�<80>�<8c>�<85>�<8d>º£�<80>�<8c>½º¿¡ ´�<99>½�<83> ¿¬°á.\n\
```

Comment:
Please check encoding, it is not **euckr**.

Reply:
Thank you. It's done.